_posts/tommytester/2022-09-22-first-post-tommytestertwo.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+author: tommytester
+title: "Tommytester's first post!"
```

Comment:
Nice YAML header! Glad you used the double quotes so that the apostrophe didn't confuse Jekyll

Reply:
Thanks!